examples/nginx-app/with-pv.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   name: nginx-deployment
   namespace: nginx-example
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
I have removed the additional notes, thanks!

My preference would be to leave the `--snapshot-volumes=false` for the basic example to not muddle the waters as Ark works just fine if you don't have PVs but the setting is enabled. If you would still like that change, I can do so.